eth/tracers/internal/tracers/call_tracer.js

```diff
@@ -208,7 +208,7 @@
 		} else if (ctx.error !== undefined) {
 			result.error = ctx.error;
 		}
-		if (result.error !== undefined) {
+		if (result.error !== undefined && (result.error !== "execution reverted" || result.output ==="0x")) {
 			delete result.output;
 		}
```

Comment:
I'm not sure why this `delete result.output` is needed at all, really. Any idea?

Reply:
Deleting this statement should cause a test-failure on `tracers_test.go`, so we'll have to change `call_tracer_revert.json`, but that seems better to me

Reply:
Regarding the `delete result.output`, I guess VM doesn't return any data before `revert(string memory message)` was added.